style.css

```diff
@@ -847,7 +847,10 @@ a:visited .nav-title,
 .widget .widget-title a,
 .widget .widget-title a:visited,
 .widget ul li a,
-.widget ul li a:visited {
+.widget ul li a:visited, 
+.site-footer .widget-area ul li a,
+.site-footer .widget-area ul li a:visited
+ {
```

Comment:
The opening brace should be on the previous line with a single space preceeding it.